wrapanapi/systems/redfish.py

```diff
@@ -4,27 +4,124 @@
 """
 from __future__ import absolute_import
 
+import redfish_client
+
+from wrapanapi.entities.server import Server, ServerState
```

Comment:
Not sure why we're listing Server and ServerState in `wrapanapi/entities/__init__.py` if we then import them directly here? :) I suggest you either go with

```python
from wrapanapi.entities import Server, ServerState
```
or just remove the two classes from `wrapanapi/entities/__init__.py` (I prefer the second option TBH).

Reply:
👍 I went for the first option, since all the others are already in the `__init__.py`